bevy_rapier3d/examples/multi_world3.rs

```diff
@@ -0,0 +1,120 @@
+use bevy::{input::common_conditions::input_just_pressed, prelude::*};
+use bevy_rapier3d::prelude::*;
+
+const N_WORLDS: usize = 2;
+
+fn main() {
+    App::new()
+        .insert_resource(ClearColor(Color::srgb(
+            0xF9 as f32 / 255.0,
+            0xF9 as f32 / 255.0,
+            0xFF as f32 / 255.0,
+        )))
+        .add_plugins((
+            DefaultPlugins,
+            RapierPhysicsPlugin::<NoUserData>::default()
+                .with_default_world(RapierContextInitialization::NoAutomaticRapierContext),
```

Comment:
Could we change `with_default_world` to `with_custom_initialization` to make it more obvious that this can be used to *not* create a default world?